build.gradle.kts

```diff
@@ -68,10 +68,11 @@ dependencies {
   kover(projects.arrowContinuations)
   kover(projects.arrowCore)
   kover(projects.arrowCoreRetrofit)
+  // kover(projects.arrowCoreSerialization)
```

Comment:
Why is this commented?

Reply:
I tried to add it for completeness, but it turns out that it causes problems with Kover (as the Optics KSP does). At the end I’ll remove both, and add a warning in the code to remember this.